warning: passing argument 1 of 'free' discards qualifiers from pointer target type

--- src/select.c.orig	Sat Nov  2 09:10:01 2013
+++ src/select.c	Sat Nov  2 09:09:51 2013
@@ -182,7 +182,7 @@ file_verdict_on (const uchar *filename)
 			   + strlen (job->file_command)
 			   + ustrlen (filename)));
   sprintf (command, "%s '%s'", job->file_command, (const char *) filename);
-  free(filename);
+  free((uchar*)filename);
   message (msg_tool, (stderr, "Reading pipe: `%s'\n", command));
   file_out = popen (command, "r");
 
